edb/server/cluster.py

```diff
@@ -245,6 +248,7 @@ async def test(timeout):
                     conn = await edgedb.async_connect(
                         host=str(self._runstate_dir),
                         port=self._effective_port,
+                        admin=True,
```

Comment:
We are deprecating and removing the `admin` argument in bindings.  Why is this necessary?

Reply:
It's not - it's just easier to make the tests pass for now, given that we still have a few `_admin_query()` calls. I can fix those to use the CLI instead of the Python binding.